webvtt_internal_test.go

```diff
@@ -13,7 +13,7 @@ func TestParseTextWebVTT(t *testing.T) {
 	t.Run("When both voice tags are available", func(t *testing.T) {
 		testData := `<v Bob>Correct tag</v>`
 
-		s := parseTextWebVTT(testData)
+		s, _ := parseTextWebVTT(testData, make([]WebVTTTag, 0, 16))
```

Comment:
The best would be to handle `nil` tags in `parseTextWebVTT` so that you can put `nil` instead of `make([]WebVTTTag, 0, 16)` here